client/src/App.js

```diff
@@ -1,23 +1,31 @@
 import React, { useState } from 'react';
 import { MuiThemeProvider } from '@material-ui/core';
-import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import {
+  BrowserRouter,
+  HashRouter,
+  Route,
+  Switch,
+  Redirect,
+} from 'react-router-dom';
 
 import { theme } from 'themes/theme';
 import { UserContext } from 'components/contexts/UserContext';
 import { FriendsContext } from 'components/contexts/FriendsContext';
 
-import Loading from 'components/Loading';
-import NavBar from 'components/NavBar';
-import LandingPage from 'pages/Landing/Landing';
-import FriendsLayout from 'pages/Friends/Friends';
-import Dashboard from 'pages/Dashboard/Dashboard';
+import NavBar from './components/NavBar';
+import Loading from './components/Loading';
+import LandingPage from './pages/Landing/Landing';
+import FriendsLayout from './pages/Friends/Friends';
+import Dashboard from './pages/Dashboard/Dashboard';
+import PollHome from './pages/Polls/PollHome';
+import Poll from './pages/Poll/Poll';
 
 import './App.css';
 
 function App() {
   const user = React.useContext(UserContext);
   const friends = React.useContext(FriendsContext);
-  const [userLoading, setUserLoading] = useState(false);
+  const [userLoading, setUserLoading] = useState(true);
```

Comment:
```suggestion
  const [userLoading, setUserLoading] = useState(false);
```

Reply:
It's one of things that came in when I merged Hakeem's branch into mine. We should make a note of it and put it in icebox for him.